internal/datanode/data_node.go

```diff
@@ -987,9 +995,11 @@ func (node *DataNode) AddImportSegment(ctx context.Context, req *datapb.AddImpor
 				statsBinLogs: req.GetStatsLog(),
 				startPos: &internalpb.MsgPosition{
 					ChannelName: req.GetChannelName(),
+					MsgID:       ids[req.GetChannelName()],
 				},
 				endPos: &internalpb.MsgPosition{
 					ChannelName: req.GetChannelName(),
+					MsgID:       ids[req.GetChannelName()],
```

Comment:
Can we also get timestamp information here?

Reply:
added